### PR TITLE
Adds Dockerfile

### PR DESCRIPTION
**Pull request by rolandpoulter**
* Switch from graphite backend to elasticsearch.
* Dockerfile for building on-statsd image.

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
 // Copyright 2015, EMC, Inc.
 
 {
-    graphitePort: 2003,
-    graphiteHost: "192.168.111.222",
+    backends: [ "statsd-elasticsearch-backend" ],
+
+    debug: true,
```
Comment (benbp) [inline lib/config.js]: Debug by default?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM roland/on-core
```
Comment (benbp) [inline Dockerfile]: Change from your fork?

Comment (benbp) [inline Dockerfile]: Oh, I see you have your own stuff in Docker hub. Perhaps we should make a RackHD user there or something?

Comment (rolandpoulter) [inline Dockerfile]: Yes, I think we can create a RackHD organization on docker hub. Then I can change it to be `RackHD/on-core`

Comment (benbp) [inline Dockerfile]: Okay. I also made a similar comment on one of Zach's PRs today. I think we should avoid putting build/assets into privately controlled repositories. @heckj see any problems with creating a RackHD org for docker hub?

File: lib/config.js (hunk 1 above)
Comment (rolandpoulter) [inline lib/config.js]: I should comment that out. :)

**Comment by heckj** [comment]
Assuming the hound comments aren't meaningful here and should be ignored.

How do the applications know where to send the statsd data to this instance? I think in the code & VM based examples, it's locked into udp on localhost - is there additional configuration needed to be passed into the other Docker files to kick that data into this container so that we collect it and route it into ElasticSearch?

Likewise, how does the configuration here for `localhost` translate into where to send the data from this statsd receiver/forwarding instance?

**Comment by rolandpoulter** [comment]
This is actually a bit magical. Since I'm loading all the RackHD docker containers with `net: "host"` they all run on the same `localhost`. That allows this to run a statsd server on port `8125` that is accessible by the other RackHD services. It takes messages from statsd and pumps them back to elasticsearch. Since I'm mapping elasticsearch port to the host in the docker compose it can receive requests from `on-statsd`.

**Comment by rolandpoulter** [comment]
Added copyright to Dockerfile

**Comment by heckj** [comment]
:+1:

**Comment by benbp** [comment]
:+1:

**Pull request merged**